### PR TITLE
Issues #SB-4805 fix: adding batchId in read course progress status api

### PR DESCRIPTION
**Pull request by manzarul**
- [ ] Have you followed [Contribution Guidelines](http://www.sunbird.org/contributions/contribution_guidelines/)
 Adding Batch id inside read course progress api.
-----

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/> Click the URL for review pull request - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/438. <br/><br/> **Modified pages are -** <br />- http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/438/apis/courseprogressapi/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/> Click the URL for review pull request - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/438. <br/><br/> **Modified pages are -** <br />- http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/438/apis/courseprogressapi/ <br />  <br/> **Modified sections are -** <br />

File: apis/courseprogressapi/courseprogressapi.yaml [hunk 1]
```diff
@@ -166,25 +166,29 @@ definitions:
...
         items:
           type: string
+      batchId: 
+       description: 'id of the batch, for which course belong to.'
+       type: string 
     required:
       - userId
       - courseIds
       - contentIds
+      - batchId
```
Comment (bvinayakumar) [inline apis/courseprogressapi/courseprogressapi.yaml]: courseIds, contentIds, batchId are optional.

File: apis/courseprogressapi/courseprogressapi.yaml [hunk 2]
```diff
@@ -166,25 +166,29 @@ definitions:
...
         items:
           type: string
       contentIds:
-        description: 'This represents the list of content IDs for that course'
+        description: 'This represents the list of content IDs for that course and batch.Pass one courseId and batchId and list of all content ids.'
         type: array
         items:
           type: string
+      batchId: 
+       description: 'id of the batch, for which course belong to.'
```
Comment (bvinayakumar) [inline apis/courseprogressapi/courseprogressapi.yaml]: ID of the batch corresponding to specified course ID.

File: apis/courseprogressapi/courseprogressapi.yaml [hunk 3]
```diff
@@ -166,25 +166,29 @@ definitions:
     type: object
     properties:
       userId:
-        description: 'This ID uniquely identifies a user'
+        description: 'This Id uniquely identifies a user'
```
Comment (bvinayakumar) [inline apis/courseprogressapi/courseprogressapi.yaml]: User Identifier

File: apis/courseprogressapi/courseprogressapi.yaml [hunk 4]
```diff
@@ -166,25 +166,29 @@ definitions:
...
-        description: 'This represents the list of course IDs'
+        description: 'This represents the list of course IDs, pass only one id incase you are passing contentIds as well. Multiple courseIds will be consider if user need to fetch data for different courseId irrespective of any batch and content.'
         example:
           - some course ID
         type: array
         items:
           type: string
       contentIds:
-        description: 'This represents the list of content IDs for that course'
+        description: 'This represents the list of content IDs for that course and batch.Pass one courseId and batchId and list of all content ids.'
```
Comment (bvinayakumar) [inline apis/courseprogressapi/courseprogressapi.yaml]: One or more content IDs corresponding to specified course ID. Specify only one course ID when specifying content IDs.

File: apis/courseprogressapi/courseprogressapi.yaml [hunk 5]
```diff
@@ -166,25 +166,29 @@ definitions:
     type: object
     properties:
       userId:
-        description: 'This ID uniquely identifies a user'
+        description: 'This Id uniquely identifies a user'
         example: Sam123
         type: string
       courseIds:
-        description: 'This represents the list of course IDs'
+        description: 'This represents the list of course IDs, pass only one id incase you are passing contentIds as well. Multiple courseIds will be consider if user need to fetch data for different courseId irrespective of any batch and content.'
```
Comment (bvinayakumar) [inline apis/courseprogressapi/courseprogressapi.yaml]: List of course IDs.

**Comment by lakhanmandloi** [comment]
Hello @manzarul,<br/><br/> _This is a auto-generated response._ <br/> Click the URL for review pull request - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/438. <br/><br/> **Modified pages are -** <br />- http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/438/apis/courseprogressapi/ <br />  <br/> **Modified sections are -** <br />

**Pull request merged**